src/components/FlyModal/FlyModal.tsx

```diff
@@ -73,7 +81,7 @@ export default class FlyModal extends React.Component<IProps> {
 					className={classnames({
 						[styles.FlyModal__HasIcon]: this.props.hasIcon,
 					})}
-					onClick={FlyModal.onRequestClose}
+					onClick={this.onRequestClose.bind(this)}
```

Comment:
This is the only thing I'd really like to see changed. I'd prefer avoid ever using bind within the `render` function as it some undesirable re-rendering implications.